translations/stripes-smart-components/en.json

```diff
@@ -217,10 +217,10 @@
   "customFields.settings.sectionTitle": "Section title",
   "customFields.settings.accordion.hidden": "(Hidden)",
   "customFields.recordAccordion.defaultName": "Custom fields",
-  "customFields.recordAccordion.required": "Field is required",
+  "customFields.recordAccordion.required": "Field is required.",
   "customFields.recordAccordion.length": "65 character limit has been exceeded. Please revise.",
   "customFields.helpText.lengthExceeded": "100 character limit has been exceeded. Please revise.",
   "customFields.fieldName.required": "Field name is required",
```

Comment:
Do we still need it?

Reply:
@Godlevskyi , we don't need it, thank you!